oidcclient/jwk_test.go

```diff
@@ -66,7 +66,8 @@ func TestProviderJSON_assertMandatoryFieldsPresent(t *testing.T) {
 func TestOIDCTenant_ReadJWKs(t *testing.T) {
 	type fields struct {
 		Duration         time.Duration
-		ZoneID           string
+		AppTID           string
```

Comment:
I dont see test case where appTid would be nil, for some very special tenanst it can happen that the appTid is not available in the token. How does the lib behaves then? At least in java lib we have implemented it in the way that if the appTid is null then we do not set any headers (neither x-app_tid nor x-clientid)